flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/ResourceManager.java

```diff
@@ -145,6 +147,12 @@
 	/** All registered listeners for status updates of the ResourceManager. */
 	private ConcurrentMap<String, InfoMessageListenerRpcGateway> infoMessageListeners;
 
+	/** The number of failed containers since the master became active. */
+	protected AtomicInteger failedContainerSoFar = new AtomicInteger(0);
+
+	/** Number of failed TaskManager containers before stopping the application. Default is  Integer.MAX_VALUE */
+	protected int maxFailedContainers = Integer.MAX_VALUE;
```

Comment:
this should be `maximumAllowedTaskManagerFailureCount`, to avoid confusion since this is only used during `closeTaskManagerConnection` but not `closeJobManagerConnection`

Reply:
Agree.